enterprise/frontend/src/metabase-enterprise/transforms/pages/JobListPage/JobList/JobList.tsx

```diff
@@ -90,6 +93,9 @@ export function JobList({ params }: { params: JobListParams }) {
                   ).format("lll")
                 : null}
             </td>
+            <td className={S.cell}>
+              <JobTransformCount jobId={job.id} />
```

Comment:
This works but do we really want to fire a request for every job on the page? It's classic N+1 behavior. 

Sure maybe we won't have many jobs but and there's at least some level of FE caching but I can't imagine these calls are cheap either (since they do the dependency graph work).

If this info is not hard to calculate, why don't we include it `TransformJob` on the BE by default?